GestureHandlerRootView.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { View } from 'react-native';
+
+export default View;
```

Comment:
can we just render children in this case? No need to introduce another view layer by default when root view isn't necessary

Reply:
I think it won't be correct to not render a view. For example, if I'm using `GestureHandlerRootView` in my library/app, I could pass some styles such as `backgroundColor`, `flexDirection` etc. If we don't return a view here, those styles will only be applied on Android.

In practice, if I'm able to pass styles here directly and make it work on both platforms, I wouldn't have to add an extra view to do it.